docker/ubi/Dockerfile

```diff
@@ -0,0 +1,45 @@
+ARG UBI_VERSION
```

Comment:
Do you think it makes sense to add `latest` as default here to make it easier to test locally? CI/CD will use the right version of course.

Reply:
Sure, we can do that. Thanks!